hstream-server/src/HStream/Server/Handler.hs

```diff
@@ -171,9 +174,18 @@ executePushQueryHandler ServerContext{..} (ServerWriterRequest meta CommandPushQ
             let sc = hstoreSourceConnector scLDClient ldreader'
             subscribeToStream sc sink Latest
             sendToClient isCancelled sc streamSend
-
-
-    Right _                              -> returnRes  "inconsistent method called"
+    Right (ShowPlan showObject) -> do
+      case showObject of
+        Streams -> do
+          names' <- try $ getLogDirectory scLDClient "/" >>= logDirectoryGetLogsName True
```

Comment:
Maybe a common prefix is better? Btw, the function `getStreams` should be added in hstore...